packages/lesswrong/server/callbacks/postCallbacks.ts

```diff
@@ -88,19 +88,6 @@ getCollectionHooks("Posts").createAfter.add((post: DbPost) => {
   }
 });
 
-/**
- * For posts created in a subforum, add the appropriate tag
- */
-getCollectionHooks("Posts").createAfter.add(async function subforumAddTag(post: DbPost, properties: CreateCallbackProperties<DbPost>) {
-  const { context } = properties
-
-  if (post.subforumTagId && context.currentUser?._id) {
-    const currentUser = context.currentUser
-    await addOrUpvoteTag({tagId: post.subforumTagId, postId: post._id, currentUser, context})
-  }
-  return post
-});
```

Comment:
No longer necessary because parent tags are auto-applied as soon as a tag is selected